sbt/src/sbt-test/project/unified/build.sbt

```diff
@@ -0,0 +1,12 @@
+lazy val root = (project in file("."))
+  .settings(
+    Global / cancelable := true,
+    ThisBuild / scalaVersion := "2.12.2",
```

Comment:
Half a day to code it and ship it. We already have all the infrastructure, we just need to do it (and it would be really simple). See [this](https://github.com/scalacenter/sbt-migration-rewrites).